### PR TITLE
Update codeFile1.ps1

### PR DESCRIPTION
**Pull request by gnowleumas**

### CODE REVIEW

**Review by sc-samuelwongchangloong — APPROVED** [review]

File: src/codeFile1.ps1 [hunk 1]
```diff
@@ -7,3 +7,4 @@ Write-Host "Hello World 6"
 Write-Host "Hello World 7"
 Write-Host "Hello World 8"
 Write-Host "Hello World 9"
+Write-Host "Hello World 10"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:10]: actually wait

**Review by sc-samuelwongchangloong — CHANGES_REQUESTED** [review]
new requirement to be addressed

File: src/codeFile1.ps1 [hunk 2]
```diff
@@ -7,3 +7,4 @@ Write-Host "Hello World 6"
 Write-Host "Hello World 7"
 Write-Host "Hello World 8"
 Write-Host "Hello World 9"
+Write-Host "Hello World 10.10"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:10]: need another 11

**Review by sc-samuelwongchangloong — CHANGES_REQUESTED** [review]

File: src/codeFile1.ps1 (hunk 2 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:10]: ok

File: src/codeFile1.ps1 [hunk 3]
```diff
@@ -8,3 +8,4 @@ Write-Host "Hello World 7"
 Write-Host "Hello World 8"
 Write-Host "Hello World 9"
 Write-Host "Hello World 10.10"
+Write-Host "Hello World 11"
```
Comment (sc-samuelwongchangloong) [inline src/codeFile1.ps1:11]: should be 11.11

**Review by sc-samuelwongchangloong — CHANGES_REQUESTED** [review]

File: src/codeFile1.ps1 (hunk 3 above)
Comment (gnowleumas) [reply src/codeFile1.ps1:11]: ok

**Review by sc-samuelwongchangloong — APPROVED** [review]

**Pull request merged**